buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterConfiguration.groovy

```diff
@@ -44,14 +45,18 @@ class ClusterConfiguration {
 
     @Input
     void plugin(String name, FileCollection file) {
-        setupCommands.put(name, ['bin/plugin', 'install', new LazyFileUri(file: file)])
+        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+            setupCommands.put(name, ['call', 'bin/plugin', 'install', new LazyFileUri(file: file)])
```

Comment:
Agree, but I would like to go one step further: The method `void plugin(String name, FileCollection file)` should just store the `<String, FileCollection>` pairs. The method `void setupCommand(String name, Object... args)` can be removed (as it is too broad and OS-dependent). This leaves it up to `ClusterFormationTasks.groovy` to implement how plugins are actually installed.